app/institutions/dashboard/-components/total-count-kpi-wrapper/component.ts

```diff
@@ -67,6 +67,26 @@ export default class TotalCountKpiWrapperComponent extends Component<TotalCountK
                 total: metrics.summaryMetrics.preprintCount,
                 icon: 'file-alt',
             },
+            {
+                title: this.intl.t('institutions.dashboard.kpi-panel.storage'),
+                total: metrics.summaryMetrics.storageByteCount,
```

Comment:
Tying back to the earlier comment, this could be `total: metrics.summaryMetrics.convertedStorageCount,`